### DISCUSSION TITLE
"On this day" functionality

### DISCUSSION THREAD

**Discussion by digitlength** [Radiokot/photoprism-android-client#40] [Ideas]
At its most basic: a notification appears from the app at the start of the day, which when clicked takes you into the app and shows you media taken on this exact day and month from previous years.

More advanced features could include custom filters (so clicking the notification only shows images, for example, or only favourites), and custom notification times.

I appreciate if this isn't a direction you're interested in!

**Comment by Radiokot** [comment]
Hey!
I like the idea, sounds doable.

**Comment by Radiokot** [comment]
Hi. If you are still interested in PhotoPrism and the gallery, here's a clone build in which you can test the Memories feature. If there is something in your library for this day in the past, the memory card is shown on the main screen. The memories are updated daily around 08:30, currently there are no notifications for them. To make the instant update, hold the search bar to get onto the dev screen and click "Update now".
https://send.bitwarden.com/#fzXZxNAsp0SmIbEjAU-wQQ/nhfMlkhEBKDrsVPn515fiA
<img width=300 src="https://github.com/Radiokot/photoprism-android-client/assets/5675681/ad282392-b6a2-4e12-9017-655a54aac8f5" />

**Comment by digitlength** [comment]
This is looking great! Two immediate bits of feedback:

"X yeas ago" should be "X years ago" (missing 'r')

For some reason, I get the collections in this order:
1 year ago
2 years ago
3
4
8
6
7

I don't have images from 5 years ago on this date so that's correct, but 8 years ago had appeared too early!

Great work on this, super cool so far!

**Comment by Radiokot** [comment]
Hi.
[It is now available as a gallery extension](https://github.com/Radiokot/photoprism-android-client/releases/tag/1.27.0(41)). Hit the "Help" button on the extension activation screen (Preferences > Extensions)  and I'll send you a key 🤝
